src/components/CompiledOutput.js

```diff
@@ -233,8 +193,12 @@ export function CompiledOutput({
                 <Segment.Group piled>{displayAvailablePresets()}</Segment.Group>
                 <Wrapper>
                   <Config
-                    value={stringConfig}
-                    onChange={handleStringConfigChange}
+                    value={
+                      babelConfig === Object(babelConfig)
+                        ? JSON.stringify(babelConfig, null, "\t")
+                        : babelConfig
```

Comment:
This might break stuff since it looks like we are passing in `babelConfig` instead of `stringConfig` to the new component